leavestracker/apps/leaves/serializers.py

```diff
@@ -8,7 +8,7 @@ class LeavesSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Leaves
-        fields = ( 'start_date','end_date','reason', 'first_name', 'last_name', 'email')
+        fields = ( 'user', 'start_date', 'end_date', 'reason', 'first_name', 'last_name', 'email')

```

Comment:
why we need to send entire user object ?

Reply:
Here user for  that we need for insert serializer data into leaves table